### PR TITLE
Remove prebuild TensorFlow plugins from DALI

### PR DESCRIPTION
**Pull request by JanuszL**
- removes building of the prebuild TF plugins in DALI as nvidia-dali-tf-plugin
  should be sufficient now
- updates documentation and adds warning about know issue with installing DALI and
  nvidia-dali-tf-plugin in the same pip command invocation

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

File: README.rst [hunk 1]
```diff
@@ -79,7 +79,7 @@ Starting DALI 0.8.0 for CUDA 10.0 based build use:
 
 .. note::
 
-  The ``nvidia-dali`` package contains prebuilt versions of the DALI TensorFlow plugin for several versions of TensorFlow. Starting DALI 0.6.1 you can also install DALI TensorFlow plugin for the currently installed version of TensorFlow, thus allowing forward compatibility:
+  Since 0.11.0  ``nvidia-dali`` package doesn't contain prebuilt versions of the DALI TensorFlow plugin, DALI TensorFlow plugin need to be installed for the currently present version of TensorFlow:
```
Comment (jantonguirao) [inline README.rst]: Needs to be installed explicitly

File: README.rst [hunk 2]
```diff
@@ -91,8 +91,9 @@ Starting DALI 0.8.0 for CUDA 10.0 based build execute:
 
    pip install --extra-index-url https://developer.download.nvidia.com/compute/redist/cuda/10.0 nvidia-dali-tf-plugin
 
+.. note::
 
-Installing this package will install ``nvidia-dali`` and its dependencies, if these dependencies are not already installed. The package ``tensorflow-gpu`` must be installed before attempting to install ``nvidia-dali-tf-plugin``.
+    Due to `know issue with installing dependent packages <https://github.com/pypa/pip/issues/1386>`_), DALI need to be installed first and then ``nvidia-dali-tf-plugin`` (two separate PIP invocations). The package ``tensorflow-gpu`` must be installed before attempting to install ``nvidia-dali-tf-plugin``.
```
Comment (jantonguirao) [inline README.rst]: Due to a

Comment (jantonguirao) [reply README.rst]: DALI needs to be installed before installing `nvidia-dali-tf-plugin` (in a separate pip install call)

Comment (JanuszL) [reply README.rst]: Done

File: README.rst (hunk 1 above)
Comment (JanuszL) [reply README.rst]: Done

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

File: README.rst [hunk 3]
```diff
@@ -91,8 +91,9 @@ Starting DALI 0.8.0 for CUDA 10.0 based build execute:
 
    pip install --extra-index-url https://developer.download.nvidia.com/compute/redist/cuda/10.0 nvidia-dali-tf-plugin
 
+.. note::
 
-Installing this package will install ``nvidia-dali`` and its dependencies, if these dependencies are not already installed. The package ``tensorflow-gpu`` must be installed before attempting to install ``nvidia-dali-tf-plugin``.
+    Due to `know issue with installing dependent packages <https://github.com/pypa/pip/issues/1386>`_), DALI needs to be installed before installing ``nvidia-dali-tf-plugin`` (in a separate pip install call). The package ``tensorflow-gpu`` must be installed before attempting to install ``nvidia-dali-tf-plugin``.
```
Comment (awolant) [inline README.rst]: Maybe `known` ?

Comment (JanuszL) [reply README.rst]: Done

**Pull request merged**